momma_dragonn/model_creators/flexible_keras.py

```diff
@@ -93,8 +93,11 @@ def _get_uncompiled_model(self, seed):
         import numpy as np
         np.random.seed(seed)
         import keras
+        # keras Graph module is deprecated in keras 1
         from keras.models import Graph
         graph = Graph()
+        #from keras.models import Model
```

Comment:
Changes look fine - can you remove stray comments from the version you actually commit?